### ISSUE TITLE
Hello, author. Do you have the labels for the test set of the INRIA Person dataset in YOLO format?

### ISSUE THREAD

**Issue by woshisad159** [WhoTHU/Adversarial_Texture#11]
Hello, author. Do you have the labels for the test set of the INRIA Person dataset in YOLO format?

**Comment by woshisad159** [comment]
I cannot download the dataset via the link provided in the README.

**Comment by WhoTHU** [comment]
Hi, we used the official INRIA Person data set. You are welcome to use any source. The test label will be created by running evaluation.py with --prepare_data as described in README. For testing, we used the labels generated by the target model on clean images. I don't have the YOLO format label for the original test labels.

**Comment by woshisad159** [comment]
Thank you very much, I understand how to proceed now. Were the training set labels generated in the same way, or were they manually re-annotated?


         原始邮件
         
       
发件人：Zhanhao Hu ***@***.***&gt;
发件时间：2026年6月29日 09:33
收件人：WhoTHU/Adversarial_Texture ***@***.***&gt;
抄送：woshisad159 ***@***.***&gt;, Author ***@***.***&gt;
主题：Re: [WhoTHU/Adversarial_Texture] Hello, author. Do you have the labels for the test set of the INRIA Person dataset in YOLO format? (Issue #11)



WhoTHU left a comment (WhoTHU/Adversarial_Texture#11)

Hi, we used the official INRIA Person data set. You are welcome to use any source. The test label will be created by running evaluation.py with --prepare_data as described in README. For testing, we used the labels generated by the target model on clean images. I don't have the YOLO format label for the original test labels.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS&nbsp;and Android. Download it today!

**Comment by WhoTHU** [comment]
We used a YOLO format of the original Inria label for training, which is from https://gitlab.com/EAVISE/adversarial-yolo/-/tree/master/yolo-labels
